class/register.js

```diff
@@ -0,0 +1,27 @@
+  // Check validation for name, mail, userId and password errors.
+  function validate() {
+    if (document.getElementsByName("nameerr")[0].innerHTML == "" && document.getElementsByName("mailerr")[0].innerHTML == "" && document.getElementsByName("usererr")[0].innerHTML == "" && document.getElementsByName("passerr")[0].innerHTML == "") {
```

Comment:
```suggestion
    if (document.getElementsByName("nameerr")[0].innerHTML == ""
       && document.getElementsByName("mailerr")[0].innerHTML == ""
       && document.getElementsByName("usererr")[0].innerHTML == ""
       && document.getElementsByName("passerr")[0].innerHTML == "") {
```

Use multiple && like this

Reply:
done